### ISSUE TITLE
Is there any easy to way to run the regression version of length of stay prediction?

### ISSUE THREAD

**Issue by EmmaRocheteau** [YerevaNN/mimic3-benchmarks#74]
Is there an easy way to run the regression set up for length of stay? I noticed that these have been reported in the nature version of the paper but I can't see an easy way to run it

**Comment by Harhro94** [comment]
Yes, if you run the `mimic3models/length_of_stay/main.py` script with the `--partition none` option, it will run the regression version.

**Comment by EmmaRocheteau** [comment]
In your nature paper you say "For length-of-stay prediction task we also tried regression models that directly predict the number of days. These models consistently performed worse than classification models in terms of kappa score, but had better mean absolute difference (MAD), as shown in Table 3. Note that the regression models for LOS prediction were trained with mean squared error loss function."

Does this mean that I need to set the loss function to mean squared error, and convert the data to days rather than hours to train the same model?

**Comment by Harhro94** [comment]
When predicting the number of days with regression the target can be fractional. Therefore, there is no need to convert data from hours to days.

In the paper the regression loss function was MSE. Note that one can replace it with MAD and get even better MAD scores.
